fairing/builders/append/append.py

```diff
@@ -73,7 +74,10 @@ def _build(self, transport, src):
         creds = docker_creds.DefaultKeychain.Resolve(src)
         with v2_2_image.FromRegistry(src, creds, transport) as src_image:
             with open(self.context_file, 'rb') as f:
-                new_img = append.Layer(src_image, f.read())
+                new_img = append.Layer(src_image, f.read(),
```

Comment:
Can you add this to the dockerfile generator as well? Or open up an issue to do it?

Reply:
Sure, I'll create an issue for this

Reply:
Actually, looks like one already exists -- #65